### PR TITLE
Extend listing search article

### PR DESCRIPTION
**Pull request by SariSaar**

### CODE REVIEW

File: src/docs/concepts-listings/how-the-search-works/index.md [hunk 1]
```diff
@@ -73,3 +108,37 @@ can be used. It takes north-east and south-west corners of a bounding
 box that limits the returned results to that area. It does not affect
 the sorting of results. The bounds can then conviniently be used to
 match search results to a map view.
+
+## How about user search?
+
+The Flex Marketplace API does not have an endpoint for querying users.
+This is because listings are modeled as the focus of the marketplace. If
```
Comment (SariSaar) [inline src/docs/concepts-listings/how-the-search-works/index.md:115]: TODO: Any other reasoning for why Marketplace API doesn't have users query?

File: src/docs/concepts-listings/how-the-search-works/index.md [hunk 2]
```diff
@@ -12,10 +12,45 @@ published: true
...
+
+- **Geolocation.** The search can be used to display only listings that
+  are within a provided radius from certain coordinates.
+- **Free text.** The search can be used to find listings that have a
+  certain keyword provided by the user. The search can find from listing
+  title and description. You can also choose to index some public data
+  fields (like listing category) so the search finds from them as well.
+- **Price.** It's possible to filter out listings with too high or too
+  low price.
+- **Availability.** It's possible to filter out listings that have
```
Comment (shareoc) [inline src/docs/concepts-listings/how-the-search-works/index.md:26]: It would be cool to have an example query of each using the SDK!

**Comment by shareoc** [comment]
Looks good!

File: src/docs/concepts-listings/how-the-search-works/index.md (hunk 2 above)
Comment (SariSaar) [reply src/docs/concepts-listings/how-the-search-works/index.md:28]: Good idea! I added examples to the ones that aren't dealt with more deeply in the second half of the article

**Pull request merged**